presto-main/src/test/java/io/prestosql/execution/TestSqlTaskExecution.java

```diff
@@ -428,7 +438,7 @@ public void testComplex(PipelineExecutionStrategy executionStrategy)
 
             //
             // test body
-            assertEquals(taskStateMachine.getState(), TaskState.RUNNING);
+            assertTrue(taskStateMachine.getState() == RUNNING || taskStateMachine.getState() == FLUSHING);
```

Comment:
why we cannot assert:
```
assertEquals(taskStateMachine.getState(), RUNNING);
```
as in `testSimple`?

Reply:
Yes you are right, I missed that change.